util/logger.js

```diff
@@ -0,0 +1,42 @@
+const miscellaneous = require('./miscellaneous');
+const mailer = require('../backend/mailer')
+BuildLog = require('../model/buildlog');
+
+/**
+ * Store the generated logs
+ * @param {Object} data : Data for storing logs
+ * @param {String} data.uniqueId : Unique id of the build
+ * @param {String} data.gitUrl : GIT URL of the repository
+ * @param {String} data.surgeDeploy : Flag to check whether it is surge deploy or not
+ * @param {string} data.surgeSuccessFlag : Flag to check whether it is deployed to surge or not
+ * @param {String} data.email : email of the user
+ * @param {Integer} data.exitCode : Exit code of the process
+ */
+exports.storeLogs = function(data, callback) {
+  if (data.exitCode === 0) {
```

Comment:
@imujjwal96 I'm not able to minimize the if case. could you suggest me a way to minimize this ?